api/src/main/java/org/apache/iceberg/transforms/Timestamps.java

```diff
@@ -127,6 +133,6 @@ public String toHumanString(Integer value) {
 
   @Override
   public String toString() {
-    return name;
+    return name + "[" + zoneOffset.getTotalSeconds() + "]";
```

Comment:
Is it possible to use `+HH:mm` or `-HH:mm` instead?

Reply:
This way look good to understand,I will change it.

`toString` return `name`, When the value of `zoneOffset.getTotalSeconds()  ` is zero . This will also be compatible with the historical version.